tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "experimentalDecorators": true,
     "jsx": "react",
     "lib": ["dom", "es2015", "es2016", "es2017"],
-    "module": "es2015",
+    "module": "esnext",
```

Comment:
Had to update this to support dynamic imports.

Reply:
Seeing as you didn’t actually end up using dynamic import right now, can this be reverted? Just to be on the safe side.